nemo/collections/nlp/models/question_answering/qa_model.py

```diff
@@ -64,7 +64,7 @@ def forward(self, input_ids, attention_mask, token_type_ids):
         if isinstance(hidden_states, tuple):
             hidden_states = hidden_states[0]
         logits = self.classifier(hidden_states=hidden_states)
-        return logits
+        return logits.float()
```

Comment:
Do we really need this conversion to float?

Reply:
Explained below.